sentence.go

```diff
@@ -6,8 +6,11 @@ import (
 )
 
 const (
-	// SentenceStart is the token to indicate the start of a sentence.
-	SentenceStart = "$"
+	// SentenceStartNormal is the token to indicate the start of a sentence.
+	SentenceStartNormal = "$"
```

Comment:
Keep the old name, it's not worth breaking backwards compatibility.

Reply:
Ok